src/main/java/calculator/Numbers.java

```diff
@@ -0,0 +1,49 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class Numbers {
+    private static final String DEFAULT_DELIMITER_REGEX = "[,:]";
+    private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.)\n(.*)");
+    private static final int DELIMITER_GROUP_INDEX = 1;
+    private static final int NUMBERS_GROUP_INDEX = 2;
+    private static final String ZERO = "0";
+
+
+    private final List<Number> numbers;
+
+    public Numbers(String text) {
+        this.numbers = textToNumbers(text);
+    }
+
+    private List<Number> textToNumbers(String text) {
+        Matcher m = CUSTOM_DELIMITER_PATTERN.matcher(text);
+        if (m.find()) {
+            String customDelimiter = m.group(DELIMITER_GROUP_INDEX);
+            return Arrays.stream(m.group(NUMBERS_GROUP_INDEX).split(customDelimiter)).map(Number::new).collect(Collectors.toList());
```

Comment:
한 줄에 점 하나만 찍어서 가독성을 높여보면 어떨까요?
```suggestion
            return Arrays.stream(m.group(NUMBERS_GROUP_INDEX)
                    .split(customDelimiter))
                    .map(Number::new)
                    .collect(Collectors.toList());
```